src/main/java/frc/robot/subsystems/drivetrain/SwerveModule.java

```diff
@@ -47,7 +47,7 @@ public class SwerveModule implements BaseSwerveModule {
     private static final double driveD = 0;
     private static final double driveFF = 0.176697057706;
 
-    private static final double steerP = 0.6;
+    private static final double steerP = 0.7;
```

Comment:
did this lower the steady state error on each module?

Reply:
Honestly not sure, I think I only had one deploy with this before the back two modules were taken off. I can check tomorrow.